components/tabstrip/tabs.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Tabs
+page_title: TabStrip Tabs
+description: Overview of the TabStrip for Blazor.
+slug: tabstrip-tabs
+tags: telerik,blazor,tab,strip,tabstrip,overview
+published: True
+position: 7
+---
+
+# Tabs
```

Comment:
should be "tab parameters"

Reply:
Updated. "Tabs Configuration" sounds better to me.